stdlib/deplinker/elf.py

```diff
@@ -0,0 +1,206 @@
+#!/usr/bin/env python3.6
+# -*- coding: utf-8 -*-
+"""A primordial dependency linker that assign requirements based on ELF dependencies."""
```

Comment:
```suggestion
"""A primary dependency linker that assign requirements based on ELF dependencies."""
```
Not sure, but "primordial" sounds a bit weird to me